docs/user/custom-domains.rst

```diff
@@ -31,39 +35,50 @@ To setup your custom domain, follow these steps:
       and for a root domain like ``example.com`` use an ANAME or ALIAS record.
 
 By default, we provide a validated SSL certificate for the domain,
-this service is provided by Cloudflare.
-The SSL certificate issuance can take about one hour,
-you can see the status of the certificate on the domain page in your project.
+managed by Cloudflare.
```

Comment:
Maybe we can link to https://www.cloudflare.com/ here

Reply:
@ericholscher not sure if you missed this or not --but there is no reaction: 👍🏼 or 👎🏼 ?

Reply:
I don't really care. We can link to them, or not. 🤷